testing/internal-integ-testing/src/main/groovy/org/gradle/test/preconditions/IntegTestPreconditions.groovy

```diff
@@ -300,11 +300,11 @@ class IntegTestPreconditions {
         }
     }
 
-    static class Jdk17FromMultipleVendors implements TestPrecondition {
+    static class JdkFromMultipleVendors implements TestPrecondition {
```

Comment:
```suggestion
    static class AlternateJdksFromMultipleVendors implements TestPrecondition {
```

🤔 The home JDK vendor could differ from that of the single available alternate, but this would return false.

Reply:
This is on purpose, perhaps it could use an even longer name or a comment. Basically, for provisioning purposes, we can't use the current JDK, because it won't trigger the provisioning.

Reply:
Right, I'm just saying the class name is a little misleading.